book/modules/appendix1.tex

```diff
@@ -351,24 +388,201 @@
+				\systeme{
+				a+3b+2c=1,
+				b+c=0,
+				3c=12
+				}
+			}
+		\end{align*}
+		The third row reveals that $c=4$; substituting into the second row, we find $b=-4$. Now we can substitute $b=-4$ and $c=4$ into the first row and we obtain $a=5$.
+		
+		The solution is $\mat{a\\b\\c}=\mat{5\\-4\\4}$. Since this is the only solution to the system, the solution set is $\Set*{\mat{5\\-4\\4}}$.
 	\end{example}
 
 	\begin{example}
-		XXX Finish. Unique solution to a 3x4 system that needs a row swap; Use back subsitution
+		Solve the system
+		\[
+			\systeme[tsr]{
+				3t+s+13r=-2,
+				t+5r=1,
+				-t+s-6r=-8,
+				t+s+4r=-6
+			}.
+		\]
+		Again, we row reduce the corresponding augmented matrix to find an equivalent system\index{System of linear equations!equivalent} that is easier to find the solution set.
+		\begin{align*}
+			{\footnotesize
+			\systeme[tsr]{
+				3t+s+13r=-2,
+				t+5r=1,
+				-t+s-6r=-8,
+				t+s+4r=-6
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				3&1&13 & -2\\
+				1&0&5 & 1\\
+				-1&1&-6 & -8\\
+				1&1&4 & -6
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_1\leftrightarrow\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				3&1&13 & -2\\
+				-1&1&-6 & -8\\
+				1&1&4 & -6
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-3\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				-1&1&-6 & -8\\
+				1&1&4 & -6
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3+\text{row}_1-\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				0&0&1 & -2\\
+				1&1&4 & -6
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_4\mapsto\text{row}_4-\text{row}_1-\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				0&0&1 & -2\\
+				0&0&1 & -2
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_4\mapsto\text{row}_4-\text{row}_3}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				0&0&1 & -2\\
+				0&0&0 & 0
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme[tsr]{
+				t+5r=1,
+				s-2r=-5,
+				r=-2,
+				0=0
+				}
+			}
+		\end{align*}
+		Our equivalent system reveals $r=-2$ which we can substitute back into the first and second rows to find that $t=11$ and $s=-9$, respectively.
+		
+		The solution vector is $\mat{t\\s\\r}=\mat{11\\-9\\-2}$ and so the solution set is $\Set*{\mat{11\\-9\\-2}}$.
 	\end{example}
 
 	In the examples so far, we've stopped row reducing when the equations became simple enough
 	to solve by inspection. However, we could continue row reducing until the system is as simple as possible.
 
 	\begin{example}
-		XXX Finish. Example 1 again, but this time put it in rref. 
+		Solve the system
+		\[
+			\systeme{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}
+		\]
+		Notice that we solved this system using a combination of row reduction and substitution in a previous example.
+		This time, let us use only row reduction and the equivalent system
```

Comment:
-`and the equivalent system`

The augmented matrix for this system is .... Based on the work from the previous example, we know ...